vscode/test/runTests.mjs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// @ts-check
+
+// Adapted from the sample at
+// https://github.com/microsoft/vscode-test-web/blob/3f0f858ab15cb65ef3c19564b0f5a6910ea9414e/sample/src/web/test/runTest.ts
+//
+// This script is run using Node.js in the dev environment. It will
+// download the latest Insiders build of VS Code for the Web and launch
+// it in a headless instance of Chromium to run the integration test suite.
+
+import { dirname, join } from "node:path";
+import { fileURLToPath } from "node:url";
+import { runTests } from "@vscode/test-web";
+
+const thisDir = dirname(fileURLToPath(import.meta.url));
+
+async function main() {
```

Comment:
If this isn't exported and is just going to be invoked immediately after, why not just write it as top level statements? (Top-level await has been supported since Node 14).

Reply:
Good point, I was copying the sample but they probably wrote it when top-level await didn't exist.